autozone-poc/src/components/single-blog/single-blog.js

```diff
@@ -0,0 +1,130 @@
+import React, { Fragment, useState } from "react";
+import ReactDOM from "react-dom";
+import Background from "../../utlis/img/bg-img/bg-7.jpg";
+import Rp1 from "../../utlis/img/bg-img/rp1.jpg";
+import Rp2 from "../../utlis/img/bg-img/rp2.jpg";
+import Rp3 from "../../utlis/img/bg-img/rp3.jpg";
+import Rp4 from "../../utlis/img/bg-img/rp4.jpg";
+import "../single-blog/single-blog.css";
+import Comments from "./Comments";
+
+//Component structure
+const Singleblog = (props) => {
```

Comment:
Looks good make it as common utlis components. So others can also use throughout the website